mysql-plugin/src/e2e-test/java/io/cdap/plugin/mysql/actions/MySQLPropertiesPageActions.java

```diff
@@ -0,0 +1,19 @@
+package io.cdap.plugin.mysql.actions;
+
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.mysql.locators.MySQLPropertiesPage;
+
+public class MySQLPropertiesPageActions {
+    public static void clickOnMysqlConnectionButton() {
+        ElementHelper.clickOnElement(MySQLPropertiesPage.connectorMysql);
+    }
+
+    public static void clickOnMySQLConnection() {
+        String connectionName = PluginPropertyUtils.pluginProp("connection.name");
+        ElementHelper.clickOnElement(MySQLPropertiesPage.MySQLConnection(connectionName));
+    }
+}
+
```

Comment:
Use 2 space indentation in this class as well. Also add package-info file for both Action and Locator class.

Reply:
DONE